pom.xml

```diff
@@ -36,6 +36,10 @@
         <api-sdk-java.version>6.0.15</api-sdk-java.version>
         <api-sdk-manager-java-library.version>3.0.5</api-sdk-manager-java-library.version>
         <api-security-java.version>2.0.3</api-security-java.version>
+
+        <!-- Jackson -->
+        <fasterxml-jackson.version>2.17.0</fasterxml-jackson.version>
```

Comment:
Is this required for tech debt? Given the issues testers are having might be worth avoiding unnecessary additions

Reply:
This is needed to override another version.